pyanaconda/ui/gui/helpers.py

```diff
@@ -300,159 +334,40 @@ def waive_clicks(self, clicks):
         """
         self._waive_clicks = clicks
 
-    @property
-    def waive_ASCII_clicks(self):
-        """Number of waive clicks the user has done to override the ASCII input check.
-
-        :returns: number of ASCII check waive clicks
-        :rtype: int
-        """
-        return self._waive_ASCII_clicks
-
-    @waive_ASCII_clicks.setter
-    def waive_ASCII_clicks(self, clicks):
-        """Set number of ASCII check waive clicks.
-
-        :param int clicks: number of ASCII check waive clicks
-        """
-        self._waive_ASCII_clicks = clicks
-
-    @property
-    def policy(self):
-        """Input checking policy.
+    def on_password_changed(self, editable, data=None):
+        """Tell checker that the content of the password field changed."""
+        self.checker.password.content = self.password
 
-        :returns: the input checking policy
-        """
-        return self._policy
-
-    @policy.setter
-    def policy(self, input_policy):
-        """Set the input checking policy.
+    def on_password_confirmation_changed(self, editable, data=None):
+        """Tell checker that the content of the password confirmation field changed."""
+        self.checker.password_confirmation.content = self.password_confirmation
 
-        :param input_policy: the input checking policy
-        """
-        self._policy = input_policy
-
-    @abstractmethod
-    def on_back_clicked(self, button):
+    def try_to_go_back(self):
         """Check whether the input validation checks allow the spoke to be exited.
 
            Unlike NormalSpoke.on_back_clicked, this function returns a boolean value.
            Classes implementing this class should run GUISpokeInputCheckHandler.on_back_clicked,
            and if it succeeded, run NormalSpoke.on_back_clicked.
         """
```

Comment:
The comment should be updated to reflect the new method name

Reply:
Fxied, thanks! :)